pkgs/tools/security/pass/extensions/audit/default.nix

```diff
@@ -22,29 +35,42 @@ in stdenv.mkDerivation rec {
   postPatch = ''
     substituteInPlace audit.bash \
       --replace-fail 'python3' "${pythonEnv.interpreter}"
-    substituteInPlace Makefile \
-      --replace-fail "install --root" "install --prefix ''' --root"
+    rm Makefile
+    patchShebangs audit.bash
   '';
 
-  outputs = [ "out" "man" ];
+  outputs = [
+    "out"
+    "man"
+  ];
+
+  build-system = with pythonPackages; [ setuptools ];
 
-  buildInputs = [ pythonEnv ];
-  nativeBuildInputs = [ makeWrapper ];
+  dependencies = [ pythonEnv ];
```

Comment:
If you use the python build system here, it's probably better to just list all the project dependencies in here, right?

Reply:
I did think about it. But this project also uses the bash script `audit.bash` which needs the whole `pythonEnv`. It seems easier to declare the `pythonEnv` and just use it (like  you had it before)